### ISSUE TITLE
Functional Requirement - Accessibility

### ISSUE THREAD

**Issue by ntdenley** [benmccamish/351visualization#17]
While animating compressions, the system should provide accessibility options to users who cannot see the animations.

**Comment by NicolasBerggren** [comment]
Some work related to this is in progress on the step_descriptions branch.
